manifests/b/BlenderFoundation/Blender/4.0.1/BlenderFoundation.Blender.installer.yaml

```
@@ -21,7 +21,7 @@ FileExtensions:
 ProductCode: '{B557A37F-811D-4A98-9306-FB34E069D999}'
 Installers:
 - Architecture: x64
-  InstallerUrl: https://www.blender.org/download/release/Blender4.0/blender-4.0.1-windows-x64.msi
+  InstallerUrl:  https://ftp.halifax.rwth-aachen.de/blender/release/Blender4.0/blender-4.0.1-windows-x64.msi

```

This points to a specific mirror, which isn’t preferred. Try using the same format as previous versions - 
```suggestion
https://download.blender.org/release/Blender4.0/blender-4.0.1-windows-x64.msi
```